src/Makefile.mk

```diff
@@ -78,9 +78,16 @@ SOURCE_DIR ?= /workspace/source
 # HELMFILE_TEMPLATE_FLAGS ?= --debug
 HELMFILE_TEMPLATE_FLAGS ?=
 
+# this option configure use of selector to regenerate only namespaces with changes
+# by default, regenerate all. To regenerate only namespace try
+# HELMFILE_USE_SELECTORS = true
+HELMFILE_USE_SELECTORS ?= false
+CLEAN_OUTPUT_DIR ?= $(if $(findstring true,$(HELMFILE_USE_SELECTORS)),,$(OUTPUT_DIR)/*/*/)
+HELMFILE_GLOBAL_FLAGS ?= $(if $(findstring true,$(HELMFILE_USE_SELECTORS)),$(shell versionStream/src/get-selectors-and-clean.sh $(OUTPUT_DIR)),)
```

Comment:
With `?=` you make it possible to override this. It is in principle reasonable that a user would like to provide their own global flags for helmfile and I don't think that should break the use of selectors. I think you can just replace `?=` with `+=` to make this work.

Reply:
of course !